<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task1windows</class>
 <widget class="QDialog" name="task1windows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>553</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>862</width>
    <height>553</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>862</width>
    <height>553</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Решение ДУ м-м Эйлера и Рунге-Кутты</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QLabel" name="lX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>x(a) =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lY">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>y(a) =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>n =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lFunc">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>y` = y * (1 - x)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lH">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>h =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_x">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_y">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_n">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_h">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="lA">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>a =</string>
   </property>
  </widget>
  <widget class="QLabel" name="lB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>b =</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_a">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ds_b">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableW">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>190</width>
     <height>441</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bEuler">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать по
методу Эйлера</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bReturn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>480</y>
     <width>110</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRG">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать по
методу Рунге-Кутты</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableW_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>191</width>
     <height>441</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
  </widget>
  <widget class="MplWidget" name="MplWidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>239</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
